apps/ledger-live-desktop/src/renderer/families/ethereum/StakeFlowModal/assets/EthStakeIllustration.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
```

Comment:
please move this file to "/apps/ledger-live-desktop/src/renderer/icons"
or "/apps/ledger-live-desktop/src/renderer/images"
we don't put assets icons in the folder ""families".

Reply:
Moved